blog/2023-09-06-fixes-and-improvements/index.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Fixes and improvements
```

Comment:
Should the title mention that these are Studio fixes and improvements?

Reply:
@mellson we don’t do updates for XState (yet) and we’re sort-of moving away from saying Studio so much in favour of just “Stately.”